### PR TITLE
[MIN-68] Update Readme to include running all pipelines locally

### PR DESCRIPTION
**Pull request by dudeperf3ct**
In this ticket, we will update the instructions in Readme required to run all the pipelines locally.

Following updates are required for `Readme.md`

- [x] Add section for running Data Scraping pipeline 
- [x] A separate section for running Data Preparation pipeline
- [x] Update embedding pipeline section to include instructions for port-forwarding the chroma server service.
- [x] Update embed_data step to use localhost as default server host instead of server.default

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,34 @@ The repository for this project is one method where you can monitor progress - w
 
 # &#127939; How do I get started?
 
-## Embedding pipeline
+## Data Scraping pipeline
+
+Before we start running the pipeline, an additional setup for creating a storage container on Azure is required to store the data. We use [DVC](https://dvc.org/doc/user-guide/data-management/remote-storage/azure-blob-storage) tool for data versioning and data management.
```
Comment (JonoCX) [inline README.md]: Before running the data scraping pipeline, ...

We use the DVC tool...

File: README.md [hunk 2]
```diff
@@ -31,7 +31,34 @@ The repository for this project is one method where you can monitor progress - w
...
+
+> Data version [documentation](docs/data-version-control.md) guides you through the process of setting up a storage container on Azure and configuring DVC.
```
Comment (JonoCX) [inline README.md]: If the user needs to go and read this as part of the setup, then it should be included in the paragraph above.

File: README.md [hunk 3]
```diff
@@ -31,7 +31,34 @@ The repository for this project is one method where you can monitor progress - w
...
+
+Run the data scraping pipeline.
```
Comment (JonoCX) [inline README.md]: Now that you're all setup, let's run the data scraping pipeline:

File: README.md [hunk 4]
```diff
@@ -31,7 +31,34 @@ The repository for this project is one method where you can monitor progress - w
...
+```
+
+In this pipeline, there are two steps:
+
+* Scrape data from Mind and NHS Mental Health websites
+* Store and version the scraped data in a storage container on Azure using DVC
+
+## Data Preparation pipeline
+
+Run the data preparation pipeline.
```
Comment (JonoCX) [inline README.md]: Now that we have data scraped, we're ready to prepare that data for the model. We've created a separate pipeline for this, where we clean, validate, and version the data...

To run the data preparation pipeline:

File: README.md [hunk 5]
```diff
@@ -31,7 +31,34 @@ The repository for this project is one method where you can monitor progress - w
...
+
+```bash
+python run.py --prepare
+```
+
+In data preparation pipeline, we clean, validate, and, version the scraped data from the data scraping pipeline. This final validated dataset is versioned and stored in a storage container on Azure using DVC.
```
Comment (JonoCX) [inline README.md]: This wants to go above the bash command, see above

File: README.md [hunk 6]
```diff
@@ -82,12 +109,22 @@ cd infrastructure/chroma_server_k8s
...
+```
+
+Finally, in a separate terminal we can run the data embedding pipeline.
 
 ```bash
-python run.py -e
+python run.py --embed
 ```
 
+In data embedding pipeline, we take the validated dataset from data preparation pipeline and use Chroma vector database to store the embedding of the text data. This pipelines uses both the Mind and NHS data.
```
Comment (JonoCX) [inline README.md]: I'd put this above the bash command, sets the scene for the user prior to them running it - they need to know what's going to happen.

**Review by JonoCX — CHANGES_REQUESTED** [review]
Nice work Shubham

**Review by osw282 — APPROVED** [review]
LGTM!

**Review by JonoCX — APPROVED** [review]

**Pull request merged**